tests/unit/algorithms/segmentation/adapters/mmseg/test_mmseg_configurer.py

```diff
@@ -280,40 +272,23 @@ def setup(self) -> None:
         self.cfg = MPAConfig.fromfile(os.path.join(DEFAULT_SEG_TEMPLATE_DIR, "semisl", "model.py"))
         data_pipeline_cfg = MPAConfig.fromfile(os.path.join(DEFAULT_SEG_TEMPLATE_DIR, "semisl", "data_pipeline.py"))
         self.cfg.merge_from_dict(data_pipeline_cfg)
-        self.cfg.model_task = "segmentation"
-        self.cfg.distributed = False
-        self.cfg.merge_from_dict(
-            MPAConfig(
-                {
-                    "data": {
-                        "train": {"otx_dataset": [], "labels": []},
-                        "val": {"otx_dataset": [], "labels": []},
-                        "test": {"otx_dataset": [], "labels": []},
-                        "unlabeled": {"otx_dataset": [0], "labels": []},
-                    }
-                }
-            )
-        )
 
     @e2e_pytest_unit
-    def test_configure_hook(self, mocker):
-        mock_ul_dataloader = mocker.patch.object(SemiSLSegmentationConfigurer, "configure_unlabeled_dataloader")
-        self.configurer.configure_hook(self.cfg)
-        mock_ul_dataloader.assert_called_once()
-
-    @e2e_pytest_unit
-    def test_configure_task(self, mocker):
-        model_cfg = ConfigDict(dict(model=dict(type="", task_adapt=True)))
-        mock_remove_hook = mocker.patch("otx.algorithms.segmentation.adapters.mmseg.configurer.remove_custom_hook")
-        self.configurer.configure_task(model_cfg)
-        mock_remove_hook.assert_called_once()
+    def test_configure_data(self, mocker):
+        mocker.patch("otx.algorithms.common.adapters.mmcv.semisl_mixin.build_dataset", return_value=True)
+        mocker.patch("otx.algorithms.common.adapters.mmcv.semisl_mixin.build_dataloader", return_value=True)
 
-    @e2e_pytest_unit
-    def test_configure_unlabeled_dataloader(self, mocker):
-        mocker_build_dataset = mocker.patch("otx.algorithms.segmentation.adapters.mmseg.configurer.build_dataset")
-        mocker_build_dataloader = mocker.patch("otx.algorithms.segmentation.adapters.mmseg.configurer.build_dataloader")
-        self.configurer.configure_hook(self.cfg)
-
-        mocker_build_dataset.assert_called_once()
-        mocker_build_dataloader.assert_called_once()
-        assert "ComposedDataLoadersHook" in [hook["type"] for hook in self.cfg.custom_hooks]
+        data_cfg = MPAConfig(
```

Comment:
What about renamed MPAConfig?
I don't think it's necessary to do this in this PR, but I thought I'd mention that MPA shouldn't be visible in OTX anymore.

Reply:
Yes there are some mpa legacy in config and logger. I think they are beyond this PR. Let's handle this in another PR, but I'm not sure when it will be ready.